src/python/pants/notes/1.13.x.rst

```diff
@@ -0,0 +1,162 @@
+1.13.x Stable Releases
+======================
+This document describes releases leading up to the ``1.13.x`` ``stable`` series.
+
+1.13.0rc0 (12/11/2018)
```

Comment:
If you look at the other notes (https://www.pantsbuild.org/notes-1.12.x.html for example), the idea is for this doc to just contain "all releases in the 1.13.x" series, without being flattened into a single release like this... sorry that that isn't clear in the docs! On the bright side, it's less work.